src/App.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Footer from "./Footer";
+// import searchButton from "./searchButton";
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* <header className="App-header">CYF Hotel</header> */}
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Footer
+        footerProps={[
```

Comment:
Do you think this is a good name for the prop? We already know that it is a prop and that it's being passed to the Footer component, so it's not really helping us understand what this prop is for.